src/helpers/getByAPI.js

```diff
@@ -258,25 +185,91 @@ export const getAllByTestId = (instance: ReactTestInstance) =>
     return results;
   };
 
+export const UNSAFE_getByType = (
+  instance: ReactTestInstance,
+  warnFn?: Function
+) =>
+  function getByTypeFn(type: React.ComponentType<any>) {
+    warnFn && warnFn('getByType');
```

Comment:
We don't need to warn now, don't we? The `UNSAFE_` is there, so I don't think it's necessary to add more to the output

Reply:
Removed warn arg and call